src/colorizer/utils/react_utils.ts

```diff
@@ -199,34 +199,50 @@ export function useScrollShadow(shadowColor: string = "#00000030"): {
 const RECENT_COLLECTIONS_STORAGE_KEY = "recentDatasets";
 const MAX_RECENT_COLLECTIONS = 10;
 
-export type RecentCollection = {
+export type StoredRecentCollection = {
   /** The absolute URL path of the collection resource. */
   url: string;
   /**
    * The user input for the collection resource.
-   * If `null`, uses the existing label (if already in recent datasets) or reuse the URL (if new).
+   * If `undefined`, uses the existing label (if already in recent datasets) or reuses the URL (if new).
```

Comment:
if undefined is allowed, should the type declaration for `label` have a question mark in it?

Reply:
Updated!